packages/kbn-test/src/kbn_client/kbn_client_requester.ts

```diff
@@ -166,7 +167,7 @@ export class KbnClientRequester {
           continue;
         }
 
-        throw new Error(`${errorMessage} -- and ran out of retries`);
+        throw new KbnClientRequesterError(`${errorMessage} -- and ran out of retries`, error);
```

Comment:
This would drop the original error, making it impossible to see the response for the failing request.